Sources/Components/BottomSheet/BottomSheetInteractiveDismissalController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public final class BottomSheetInteractiveDismissalController: UIPercentDrivenInteractiveTransition {
+    let dismissalTransitioningRect: CGRect
```

Comment:
Could those 4 properties be private? or they're used somewhere outside of this class?

Reply:
Good point! I should take some time to go through the access control.